rare/components/tabs/games/game_info/game_info.py

```diff
@@ -104,13 +109,14 @@ def repair(self):
         if not os.path.exists(repair_file):
             QMessageBox.warning(
                 self,
-                "Warning",
+                self.tr("Warning - {}").format(self.igame.title),
                 self.tr(
-                    "Repair file does not exist or game does not need a repair. Please verify game first"
+                    "Repair file does not exist or game does not need a repair. "
+                    "Please verify the game first"
```

Comment:
I guess you are aware of this breaking translations? (note `verify game` vs. `verify the game`)

Reply:
Yes, but we do not have and English translation, so it is a must fix here.